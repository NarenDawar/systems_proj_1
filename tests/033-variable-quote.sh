
echo "Hello your language is ${LANG}"

echo 'Hello your language is ${LANG}'
