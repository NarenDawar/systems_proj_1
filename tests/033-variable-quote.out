Hello your language is en_US.UTF-8
Hello your language is ${LANG}
